src/main/java/edu/greenblitz/robotName/Robot.java

```diff
@@ -26,13 +27,12 @@ public enum RobotType {
     @Override
     public void robotInit() {
         initializeLogger();
-
+        CommandScheduler.getInstance().enable();
+        Dashboard.init();
         SwerveChassis.init();
         SwerveChassis.getInstance().setDefaultCommand(new MoveByJoysticks(MoveByJoysticks.DriveMode.NORMAL));
         SwerveChassis.getInstance().resetAllEncoders();
```

Comment:
remove the SwerveChassis.init() and everything related to it.it might cause problems when the code wouldn't find the relevant motors.